src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -0,0 +1,476 @@
+                interContigPair.serialize(kryo, output);
+            }
+
+            @Override
+            public InterContigPair read( final Kryo kryo, final Input input, final Class<InterContigPair> klass ) {
+                return new InterContigPair(kryo, input);
+            }
+        }
+    }
+
+    @DefaultSerializer(OutiesPair.Serializer.class)
+    public static final class OutiesPair extends BreakpointEvidence {
+        OutiesPair( final GATKRead read, final ReadMetadata metadata ) {
+            super(read, metadata);
+        }
+
+        private OutiesPair( final Kryo kryo, final Input input ) { super(kryo, input); }
+
+        @Override
+        public boolean equals( final Object obj ) {
+            return obj instanceof OutiesPair && compareTo((OutiesPair)obj) == 0;
+        }
+
+        @Override
+        public String toString() {
+            return super.toString() + " OutiesPair";
+        }
+
+        public static final class Serializer extends com.esotericsoftware.kryo.Serializer<OutiesPair> {
+            @Override
+            public void write( final Kryo kryo, final Output output, final OutiesPair mateUnmapped ) {
+                mateUnmapped.serialize(kryo, output);
+            }
+
+            @Override
+            public OutiesPair read( final Kryo kryo, final Input input, final Class<OutiesPair> klass ) {
+                return new OutiesPair(kryo, input);
+            }
+        }
+    }
+
+    @DefaultSerializer(SameStrandPair.Serializer.class)
+    public static final class SameStrandPair extends BreakpointEvidence {
+        SameStrandPair( final GATKRead read, final ReadMetadata metadata ) {
+            super(read, metadata);
+        }
+
+        private SameStrandPair( final Kryo kryo, final Input input ) { super(kryo, input); }
+
+        @Override
+        public boolean equals( final Object obj ) {
+            return obj instanceof SameStrandPair && compareTo((SameStrandPair)obj) == 0;
+        }
+
+        @Override
+        public String toString() {
+            return super.toString() + " SameStrandPair";
+        }
+
+        public static final class Serializer extends com.esotericsoftware.kryo.Serializer<SameStrandPair> {
+            @Override
+            public void write( final Kryo kryo, final Output output, final SameStrandPair sameStrandPair ) {
+                sameStrandPair.serialize(kryo, output);
+            }
+
+            @Override
+            public SameStrandPair read( final Kryo kryo, final Input input, final Class<SameStrandPair> klass ) {
+                return new SameStrandPair(kryo, input);
+            }
+        }
+    }
+
+    @DefaultSerializer(WeirdTemplateSize.Serializer.class)
+    public static final class WeirdTemplateSize extends BreakpointEvidence {
+        private final int templateSize;
+
+        WeirdTemplateSize( final GATKRead read, final ReadMetadata metadata ) {
```

Comment:
Agreed that we don't need to fix it right now for this PR but it'd be good if you could put in a TODO comment at the place where it's disabled and create a github issue (you can label it "SV") saying it should be turned on eventually. And if you happen to have any numbers about the problem or other thoughts you could throw them in the ticket.